helper/testhelpers/testhelpers.go

```diff
@@ -305,20 +294,30 @@ func GetPerfReplicatedClusters(t testing.T, conf *vault.CoreConfig, opts *vault.
 	// Set this lower so that state populates quickly to standby nodes
 	cluster.HeartbeatInterval = 2 * time.Second
 
-	opts1 := *opts
-	opts1.Logger = logger.Named("perf-pri")
-	ret.PerfPrimaryCluster, _ = ConfClusterAndCore(t, conf, &opts1)
+	localopts := *opts
+	localopts.Logger = logger.Named("perf-pri")
+	ret.PerfPrimaryCluster, _ = ConfClusterAndCore(t, conf, &localopts)
 
-	opts2 := *opts
-	opts1.Logger = logger.Named("perf-sec")
-	ret.PerfSecondaryCluster, _ = ConfClusterAndCore(t, conf, &opts2)
+	localopts.Logger = logger.Named("perf-sec")
+	numCores := opts.NumCores
+	if numCores == 0 {
```

Comment:
Shouldn't this be called before https://github.com/hashicorp/vault/pull/6855/files#diff-56dc57f8861cd469610d671d266b7a47R299 (`ret.PerfPrimaryCluster, _ = ConfClusterAndCore(t, conf, &localopts)`)

Reply:
I'm not sure what you're saying, can you rephrase / expand?

Reply:
Shouldn't the ordering be similar to https://github.com/hashicorp/vault/pull/6855/files#diff-56dc57f8861cd469610d671d266b7a47R330, i.e. it should go before the first `ConfClusterAndCore` call.

Reply:
We don't need to know how many cores are used for the first cluster, as it simply assigns them sequentially.  We need to know for subsequent clusters so we can properly increment firstCoreNumber.